RealmSwift-swift2.0/List.swift

```diff
@@ -405,7 +404,8 @@ extension List: RealmCollectionType, RangeReplaceableCollectionType {
     - parameter subRange:    The range of elements to be replaced.
     - parameter newElements: The new elements to be inserted into the list.
     */
-    public func replaceRange<C: CollectionType where C.Generator.Element == T>(subRange: Range<Int>, with newElements: C) {
+    public func replaceRange<C: CollectionType where C.Generator.Element == T>(subRange: Range<Int>,
+        with newElements: C) {
```

Comment:
The problem isn't where it's split, but rather that after splitting it's indented to the same place as the body of the function.